srcpkgs/ncspot/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.11.2
+revision=2
+build_style=cargo
+hostmakedepends="pkg-config python3 libpulseaudio"
+makedepends="openssl-devel python3-devel libxcb-devel ncurses-devel dbus-devel"
+short_desc="ncurses Spotify client"
+maintainer="d33vliter <d33vliter@protonmail.com>"
+license="BSD 2-Clause License"
+homepage="https://github.com/hrkfdn/ncspot"
+distfiles="https://github.com/hrkfdn/ncspot/archive/refs/tags/v${version}.tar.gz"
+checksum=0fd9696e13912b906605e9972bf3b2a8a68688cce17478353713550801c94fab
+
+do_install() {
+	cargo build --release
+}
```

Comment:
```suggestion
```
the default do_install from the cargo build_style should work fine here. does it not?

Reply:
https://github.com/void-linux/void-packages/actions/runs/3792212763/jobs/6448498462#step:8:9

do_build is already building the binaries, and do_install is when those files are moved to the package destination directory. with do_install overridden, it creates an empty package